v2/stacks/feedback/deps.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  mailhog:
+    extends:
+      file: ${PATH_MANIFESTS}/deps/mailhog.yml
+      service: mailhog
+  http-echo:
+    extends:
+      file: ${PATH_MANIFESTS}/deps/http-echo.yml
+      service: http-echo
```

Comment:
Could put comment here to explain this is not part of the service, but is to stub out health checks.

Reply:
Done